docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -161,48 +193,120 @@ oc expose svc/greeting <3>
 oc get routes <4>
 curl http://<route>/hello <5>
 ----
-<1> Lists the image streams created. The image stream of our application should be tagged as <project>/openshift-quickstart:1.0.0-SNAPSHOT.
+<1> Lists the image streams created.
+The image stream of the application should be tagged as <project>/openshift-quickstart:1.0.0-SNAPSHOT.
 <2> Create a new application from the image source.
 <3> Expose the service to the outside world.
 <4> Get the list of exposed routes.
 <5> Access your application.
 
-After this setup the next time the container image is built a deployment to OpenShift is triggered automatically.
-In other words, you don't need to repeat the above steps.
+After this setup, the next time you build a container image, a deployment to OpenShift is triggered automatically so you do not need to repeat the above steps.
+
+=== Option: Configure OpenShift application manually by using the Docker build
```

Comment:
Hi @iocanel, Thank you for your additional review and I updated the sections per your feedback. Following our meeting, I also updated the _Choose how you want to build and deploy your Quarkus application_ chapter to add the Docker build strategy as a subsection (content backported from downstream guide). Does this look okay to you?